build/variables/build.yml

```diff
@@ -1,5 +1,5 @@
 variables:
-  DotNet.Sdk.Version: '3.1.201'
+  DotNet.Sdk.Version: '6.0.100'
```

Comment:
Are we doing this together with the implementation of the eventgrid subscription ?
Should'nt we do this in a separate PR ?

Reply:
Yes, I wanted to split it too, but this PR also adds two new projects, and it would need to still use the 'old' .NET Core for that. Seemed a bit an extra, unnecessary step.

Reply:
But, we can always take a step back, and only focus on the features here. 
It makes me wonder, though, if the migration towards .NET 6 should be considered a breaking change across all repo's. and/or if we should double support both .NET Core and .NET 6....

Reply:
Voila, removed this updates from this PR so it only contains the feature update. That's better indeed. Let's do this .NET 6 update separately. 👍